index.js

```diff
@@ -7,88 +7,12 @@ function ask(questionText) {
   });
 }
 
-
 start();
 
 async function start() {
-  console.log("\nLet's play a game where you make up a number and I try to guess it.")
-  let secretNumber = await ask("\nWhat is your secret number?\nI won't peek, I promise...\n");
+  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
+  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
   console.log('You entered: ' + secretNumber);
-
-  function compGuess(max, min) {
-
-    return (min) + Math.floor(Math.random() * (max - min))
-
-  }
-  game()
-  async function game() {
-    let max = 100
-    let min = 1
-    let turns = 0;
-
-    console.log("\nOkay computer give it your best shot!\nGuess a number between 1 and 100!")
-    let guess = compGuess(max, min)
-    let playerAnswer = await ask("Is your number " + guess + "?")
-
-    while (playerAnswer !== "yes") {
-      guess = compGuess(max, min)
-      playerAnswer = await ask("Is your number " + guess + "?")
-
-      turns = turns + 1
```

Comment:
If you put this code below the if... else if.. else logic block the game won't ask the number twice at the start before going into the higher/lower logic.